proto/p2p/p2p.proto

```diff
@@ -61,297 +61,357 @@ message Message {
+// discover each other.
 //
-// On receiving "peer_list", the engine starts/updates the tracking information
-// of the remote peer.
+// PeerListAck should be sent in response to a PeerList.
 message PeerList {
   repeated ClaimedIpPort claimed_ip_ports = 1;
 }
 
-// "peer_ack" is sent in response to a "peer_list" message. The "tx_id" should
-// correspond to a "tx_id" in the "peer_list" message. The sender should set
-// "timestamp" to be the latest known timestamp of a signed IP corresponding to
-// the nodeID of "tx_id".
-//
-// Upon receipt, the "tx_id" and "timestamp" will determine if the receiptent
-// can forgo future gossip of the node's IP to the sender of this message.
+// PeerAck acknowledges that a gossiped peer in a PeerList message will be
+// tracked by the remote peer.
 message PeerAck {
+  // P-Chain transaction that added the acknowledged peer to the validator
+  // set
   bytes tx_id = 1;
+  // Timestamp of the signed ip of the peer
   uint64 timestamp = 2;
 }
 
-// Message that responds to a peer_list message containing the AddValidatorTxIDs
-// from the peer_list message that we currently have in our validator set.
+// PeerListAck is sent in response to PeerList to acknowledge the subset of
+// peers that the peer will attempt to connect to.
 message PeerListAck {
   reserved 1; // deprecated; used to be tx_ids
-
   repeated PeerAck peer_acks = 2;
 }
 
+// GetStateSummaryFrontier requests a peer's most recently accepted state
+// summary
 message GetStateSummaryFrontier {
+  // Chain being requested from
   bytes chain_id = 1;
+  // Unique identifier for this request
   uint32 request_id = 2;
+  // Timeout (ns) for this request
   uint64 deadline = 3;
 }
 
+// StateSummaryFrontier is sent in response to a GetStateSummaryFrontier request
 message StateSummaryFrontier {
+  // Chain being responded from
   bytes chain_id = 1;
+  // Request id of the original GetStateSummaryFrontier request
   uint32 request_id = 2;
+  // The requested state summary
   bytes summary = 3;
 }
 
+// GetAcceptedStateSummary requests a set of state summaries at a set of
+// block heights
 message GetAcceptedStateSummary {
+  // Chain bein requested from
   bytes chain_id = 1;
+  // Unique identifier for this request
   uint32 request_id = 2;
+  // Timeout (ns) for this request
   uint64 deadline = 3;
+  // Heights being requested
   repeated uint64 heights = 4;
 }
 
+// AcceptedStateSummary is sent in response to GetAcceptedStateSummary
 message AcceptedStateSummary {
+  // Chain being responded from
   bytes chain_id = 1;
+  // Request id of the original GetAcceptedStateSummary request
   uint32 request_id = 2;
+  // State summary ids
   repeated bytes summary_ids = 3;
 }
 
+// The consensus engine that should be used when handling a consensus request.
 enum EngineType {
   ENGINE_TYPE_UNSPECIFIED = 0;
+  // Only the X-Chain uses avalanche consensus. Post linearization, this is
+  // no longer used.
```

Comment:
This is still used for bootstrapping the DAG. We should be able to remove this from all the `Get` + `Put` + `PullQuery` + `PushQuery` + `Chits` messages. We also don't _really_ need this for the `GetAcceptedFrontier` and `GetAccepted` messages. (But removing these isn't as easy as just removing them from the protos unfortunately) 

We really only need it in `GetAncestors` now that the X-chain is linearized.

Reply:
Removed the second sentence